pkgs/games/dwarf-fortress/wrapper/default.nix

```diff
@@ -41,7 +43,11 @@ in
 stdenv.mkDerivation rec {
   name = "dwarf-fortress-${dwarf-fortress-original.dfVersion}";
 
-  compatible = lib.all (x: assert (x.dfVersion == dwarf-fortress-original.dfVersion); true) pkgs;
+  compatible = lib.all (x: x.dfVersion == dwarf-fortress-original.dfVersion) [
+    dwarf-fortress-original
+    dwarf-fortress-unfuck
+    dfhack
+  ];
```

Comment:
Not sure if this is right but it was complaining about not being to find dfVersion in all of the arguments

Reply:
Yeah probably there is a dfVersion somewhere that needs to be updated.